resources/js/components/UserEditForm.vue

```diff
@@ -315,6 +315,10 @@ export default {
           console.log('success');
         })
         .catch(response => {
+          if (response.response.status === 422) {
```

Comment:
To avoid issues with `null` values you probably want to write this as `response && response.response && response.response.status === 422` as I have learned from Typescript bugging me about not doing that many a time

Also, can you rename the parameter `response` to something else?